2-multitenant/modules/env_baseline/wi.tf

```diff
@@ -12,20 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-resource "google_service_account" "bank_of_anthos" {
+resource "google_service_account" "app_service_accounts" {
+  for_each = var.apps
+
   project                      = local.cluster_project_id
-  account_id                   = "bank-of-anthos"
-  display_name                 = "bank-of-anthos"
+  account_id                   = each.key
+  display_name                 = each.key
   create_ignore_already_exists = true
 }
 
+#TODO: Probably move to AppInfra
 resource "google_service_account_iam_binding" "workload_identity" {
-  service_account_id = google_service_account.bank_of_anthos.name
+  for_each = google_service_account.app_service_accounts
+
+  service_account_id = each.value.id
   role               = "roles/iam.workloadIdentityUser"
 
   members = [
-    "serviceAccount:${local.cluster_project_id}.svc.id.goog[accounts-${var.env}/bank-of-anthos]",
-    "serviceAccount:${local.cluster_project_id}.svc.id.goog[ledger-${var.env}/bank-of-anthos]",
+    "serviceAccount:${local.cluster_project_id}.svc.id.goog[accounts-${var.env}/${each.value.display_name}]",
+    "serviceAccount:${local.cluster_project_id}.svc.id.goog[ledger-${var.env}/${each.value.display_name}]",
```

Comment:
For a future PR: with BYOID, I had hoped we could delegate creation of KSAs to the app operator.  That means that this list is not part of a baseline, but instead different per namespace.